fsspec/implementations/cached.py

```diff
@@ -438,9 +437,6 @@ def __getattribute__(self, item):
             "cache_size",
             "pipe_file",
             "pipe",
-            "isdir",
```

Comment:
These ones, plus `_mkcache` above should very likely be kept in this list - the caching filesystem can care about intercepting the target filesystem for these operations. On what basis are they being removed?

Reply:
They are duplicate items of a set.